packages/angular_devkit/build_angular/package.json

```diff
@@ -26,24 +27,25 @@
     "loader-utils": "1.2.3",
     "mini-css-extract-plugin": "0.5.0",
     "minimatch": "3.0.4",
-    "parse5": "4.0.0",
     "opn": "5.4.0",
+    "parse5": "4.0.0",
     "postcss": "7.0.14",
     "postcss-import": "12.0.1",
     "postcss-loader": "3.0.0",
     "raw-loader": "1.0.0",
     "rxjs": "6.4.0",
     "sass-loader": "7.1.0",
     "semver": "5.6.0",
-    "source-map-support": "0.5.10",
     "source-map-loader": "0.2.4",
-    "speed-measure-webpack-plugin": "1.3.0",
+    "source-map-support": "0.5.11",
+    "speed-measure-webpack-plugin": "1.3.1",
     "stats-webpack-plugin": "0.7.0",
     "style-loader": "0.23.1",
     "stylus": "0.54.5",
     "stylus-loader": "3.0.2",
-    "tree-kill": "1.2.1",
     "terser-webpack-plugin": "1.2.3",
+    "tree-kill": "1.2.1",
+    "typescript": "^3.3.4000",
```

Comment:
TypeScript should be a peerDependency and we need to also support older versions of TS
```
    "typescript": ">=2.7 < 3.4",
```

Reply:
+1